adafruit_platformdetect/board.py

```diff
@@ -349,6 +351,15 @@ def _stm32mp1_id(self) -> Optional[str]:
             return boards.OSD32MP1_BRK
         if "OSD32MP1-RED" in board_value:
             return boards.OSD32MP1_RED
+        if "STM32MP1XX OLinuXino" in board_value:
+            return boards.STMP157 
```

Comment:
Please be more specific on the board name. STMP157 covers a whole bunch of boards and this is meant to be an identifier for the specific board. Something like STMP157_OLINUXINO_LIME2 would be a lot more useful and specific.

Reply:
The value inside the /proc/device-tree/model file for this board is:
`STM32MP1XX OlinuXino^@`
this is why i used it

Reply:
Thanks